InvoiceWorker/Models/Invoice.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace InvoiceWorker.Models
+{
+    public class Invoice
+    {
+        public Guid InvoiceId { get; init; }
+        public string InvoiceNumber { get; init; }
+        public IEnumerable<InvoiceLineItem> LineItems { get; init; }
```

Comment:
Should we have IEnumerable as a concrete implementation (e.g. list) to see if they like it?